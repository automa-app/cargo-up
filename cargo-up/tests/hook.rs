use insta::assert_snapshot;
use serial_test::serial;
mod utils;

#[test]
#[serial]
fn test_hook_on() {
    let (out, err) = utils::run_upgrader("hook_on", "0.6.0", true);
    assert_snapshot!(out);
    assert_snapshot!(err);
}
