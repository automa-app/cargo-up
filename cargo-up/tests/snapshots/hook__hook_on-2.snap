---
source: tests/hook.rs
expression: err
---


Pre loading upgradee ... done

